packages/core/package.json

```diff
@@ -37,15 +37,18 @@
     "emotion-server": "^10.0.27",
     "emotion-theming": "^10.0.27",
     "html-tag-names": "^1.1.2",
-    "react": "^16.5.2",
+    "react": "17.0.0-rc.0",
     "svg-tag-names": "^1.1.1"
   },
   "repository": "https://github.com/emotion-js/emotion/tree/master/packages/core",
   "publishConfig": {
     "access": "public"
   },
-  "umd:main": "dist/core.umd.min.js",
   "preconstruct": {
-    "umdName": "emotionCore"
+    "source": "src/index.js",
+    "entrypoints": [
```

Comment:
Just followed the instructions in https://preconstruct.tools/guides/adding-a-second-entrypoint

The UMD build for `jsx-runtime` failed on a "missing external" for `react/jsx-runtime`. This makes sense, but nothing I tried worked to fix it. My expectation was that adding the following to preconstruct would get things working

```js
"globals": {
  "react/jsx-runtime": "JSXRuntime"
}
```
would fix it, but it didn't. Since umd has to be on for all entrypoints or none, I disabled it for `@emotion/core` to get things working.